src/core/proj/qgscoordinatetransform.cpp

```diff
@@ -634,9 +634,9 @@ QgsRectangle QgsCoordinateTransform::transformBoundingBox( const QgsRectangle &r
 
   // We're interfacing with C-style vectors in the
   // end, so let's do C-style vectors here too.
-  QVector<double> x( nXPoints * nYPoints );
-  QVector<double> y( nXPoints * nYPoints );
-  QVector<double> z( nXPoints * nYPoints );
+  std::vector<double> x( nXPoints * static_cast< std::size_t >( nYPoints ) );
```

Comment:
the comment about C-style vector above is (was) a bit misleading :-)

Reply:
Ok, killing it :)